tests/build-gating.fmf

```diff
@@ -0,0 +1,48 @@
+#
+# Snapshot gating tests for all *LLVM* packages in the mix
+#
+# Compatible with various these distributions:
+#
+#   * Fedora
+#
+
+summary: LLVM Tests for snapshot gating
+discover:
+    - name: clang-tests
+      how: fmf
+      url: https://src.fedoraproject.org/tests/clang.git
+      ref: main
+      filter: "tag:-spoils-installation & tag:-not-in-default"
+    - name: compiler-rt-tests
+      how: fmf
+      url: https://src.fedoraproject.org/tests/compiler-rt.git
+      ref: main
+      filter: "tag:-spoils-installation & tag:-not-in-default"
+    - name: libomp-tests
+      how: fmf
+      url: https://src.fedoraproject.org/tests/libomp.git
+      ref: main
+      filter: "tag:-spoils-installation & tag:-not-in-default"
+    - name: upstream-llvm-integration-testsuite
```

Comment:
This discover config only gets integration-test-suite from LLVM tests. If you want to run all the LLVM tests I'd suggest to remove [this line](https://github.com/kwk/llvm-daily-fedora-rpms/pull/74/files#diff-48fcdeadae4f302e365808cdd08ee1cd02b46c59817c4269207104c7035cf935R30) (`test: integration-test-suite`), and maybe rename this discover config to "llvm-tests".

Reply:
I've seen this to be special but I wasn't sure (and frankly, didn't check) how the llvm repo tests this. But you're right. If I can test everything, then I want to do that. Done in 3592d34.